__tests__/file-spec.js

```
@@ -257,7 +255,8 @@ describe('File', () => {
           expect(err).toBe(testContext.error);
           expect(file).toBe(undefined);
           done();
-        });
+        })
+        .catch(() => {});
```

What's the purpose of this empty catch?